detector/aws_instance_invalid_type.go

```diff
@@ -6,7 +6,15 @@ import (
 	"github.com/wata727/tflint/issue"
 )
 
-func (d *Detector) DetectAwsInstanceInvalidType(issues *[]*issue.Issue) {
+type AwsInstanceInvalidTypeDetector struct {
+	*Detector
+}
+
+func (d *Detector) CreateAwsInstanceInvalidTypeDetector() *AwsInstanceInvalidTypeDetector {
+	return &AwsInstanceInvalidTypeDetector{d}
+}
+
+func (d *AwsInstanceInvalidTypeDetector) Detect(issues *[]*issue.Issue) {
```

Comment:
__[gometalinter]__
* [golint] exported method AwsInstanceInvalidTypeDetector.Detect should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->